awsiot/greengrasscoreipc/model.py

```diff
@@ -3371,24 +3861,575 @@ def set_shadow_name(self, shadow_name: str):
+        self.certificate_sha1_hash = certificate_sha1_hash  # type: typing.Optional[str]
+
+    def set_password(self, password: str):
+        self.password = password
+        return self
+
+    def set_username(self, username: str):
+        self.username = username
+        return self
+
+    def set_password_expiration(self, password_expiration: datetime.datetime):
+        self.password_expiration = password_expiration
+        return self
+
+    def set_certificate_sha256_hash(self, certificate_sha256_hash: str):
+        self.certificate_sha256_hash = certificate_sha256_hash
+        return self
+
+    def set_certificate_sha1_hash(self, certificate_sha1_hash: str):
+        self.certificate_sha1_hash = certificate_sha1_hash
+        return self
+
+
+    def _to_payload(self):
+        payload = {}
+        if self.password is not None:
+            payload['password'] = self.password
+        if self.username is not None:
+            payload['username'] = self.username
+        if self.password_expiration is not None:
+            payload['passwordExpiration'] = self.password_expiration.timestamp()
+        if self.certificate_sha256_hash is not None:
+            payload['certificateSHA256Hash'] = self.certificate_sha256_hash
+        if self.certificate_sha1_hash is not None:
+            payload['certificateSHA1Hash'] = self.certificate_sha1_hash
+        return payload
+
+    @classmethod
+    def _from_payload(cls, payload):
+        new = cls()
+        if 'password' in payload:
+            new.password = payload['password']
+        if 'username' in payload:
+            new.username = payload['username']
+        if 'passwordExpiration' in payload:
+            new.password_expiration = datetime.datetime.fromtimestamp(payload['passwordExpiration'], datetime.timezone.utc)
+        if 'certificateSHA256Hash' in payload:
+            new.certificate_sha256_hash = payload['certificateSHA256Hash']
+        if 'certificateSHA1Hash' in payload:
+            new.certificate_sha1_hash = payload['certificateSHA1Hash']
+        return new
+
+    @classmethod
+    def _model_name(cls):
+        return 'aws.greengrass#CreateDebugPasswordResponse'
+
+    def __repr__(self):
+        attrs = []
+        for attr, val in self.__dict__.items():
+            if val is not None:
+                attrs.append('%s=%r' % (attr, val))
+        return '%s(%s)' % (self.__class__.__name__, ', '.join(attrs))
+
+    def __eq__(self, other):
+        if isinstance(other, self.__class__):
+            return self.__dict__ == other.__dict__
+        return False
+
+
+class CreateDebugPasswordRequest(rpc.Shape):
+    """
+    CreateDebugPasswordRequest
+    """
+
+    def __init__(self):
+        super().__init__()
+
+
+    def _to_payload(self):
+        payload = {}
+        return payload
+
+    @classmethod
+    def _from_payload(cls, payload):
+        new = cls()
+        return new
```

Comment:
Are these supposed to return empty data?

Reply:
The `CreateDebugPasswordRequest` has no fields, so yes, this is accurate.